### PR TITLE
Surround `commits` under try-except block

### PR DESCRIPTION
**Pull request by validcube**
## Description

Fix: Random CI Issue with send notification

## Type of change
:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
✅ Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: ci.py [hunk 1]
```diff
@@ -45,8 +45,12 @@ def send_artifact_to_telegram_chat(chat_id):
 def send_internal_notifications():
     repository = git.Repo(".")
     commit_range = f"{github_event_before}...{github_sha}"
-    commits = list(repository.iter_commits(commit_range))
-
+    try:
+        commits = list(repository.iter_commits(commit_range))
+    except git.exc.GitCommandError as error:
+        print(f"Error fetching commits: {e}")
```
Comment (validcube) [inline ci.py]:
```suggestion
        print(f"Error fetching commits: {error}")
```

**Review by SuperDragonXD — APPROVED** [review]
Thanks! (I was planning to implement this, but I didn't have time to do it.)

**Comment by Goooler** [comment]
@validcube Thanks! Can you patch some changes like this to lawnicon?

**Comment by validcube** [comment]
> @validcube Thanks! Can you patch some changes like this to lawnicon?

🥞 Acknowledge, will do it after I finished repairing my pc. (estimated time should be ~1d)

**Pull request merged**